meta-lmp-base/conf/distro/include/lmp.inc

```diff
@@ -56,8 +56,7 @@ USERADD_GID_TABLES = "files/lmp-group-table"
 USERADD_ERROR_DYNAMIC = "error"
 
 # Default image formats
-IMAGE_FSTYPES = "wic wic.gz wic.bmap"
-IMAGE_FSTYPES_remove = "tar.gz tar.bz2"
```

Comment:
Remove itself should be fine, this was more to avoid generating artifacts that are not required at the base level, but we can easily move to lmp-machine-custom and have it be bsp specific.